### PR TITLE
fix(statutory): add information on how to withdraw your application

### PR DESCRIPTION
**Pull request by LeonVreling**

### CODE REVIEW

File: src/views/statutory/EditCandidate.vue [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="tile is-ancestor ">
     <div class="tile is-child">
+      <span class="notification is-info" v-if="$route.params.candidate_id">
+        To redraw your candidature, you should send an email to the Juridical Commission via <u>jc@aegee.eu</u>.
```
Comment (WikiRik) [inline src/views/statutory/EditCandidate.vue]:
```suggestion
        To withdraw your candidature, you should send an email to the Juridical Commission via <u>jc@aegee.eu</u>.
```

Comment (WikiRik) [reply src/views/statutory/EditCandidate.vue]: And maybe a mailto:

Comment (LeonVreling) [reply src/views/statutory/EditCandidate.vue]: Hahah yess that is what I wanted to type, oops

File: src/views/statutory/EditCandidate.vue [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <template>
   <div class="tile is-ancestor ">
     <div class="tile is-child">
+      <span class="notification is-info" v-if="$route.params.candidate_id">
+        To withdraw your candidature, you should send an email to the <a href="mailto:jc@aegee.eu">Juridical Commission</a>.
```
Comment (WikiRik) [inline src/views/statutory/EditCandidate.vue]:
```suggestion
        To withdraw your candidature, you should send an email to the <a href="mailto:juridical@aegee.eu">Juridical Commission</a>.
```
I think both work, but juridical is also the one we use in statutory

**Review by WikiRik — APPROVED** [review]

**Pull request merged**